packages/algorand-js/src/lib/helpers.ts

```diff
@@ -31,6 +31,40 @@ export function assertOnlyDigits (val: string): void {
  */
 export function assertLen (val: number, expected: number, line: number): void {
   if (val !== expected) {
-    throw new TealError(ERRORS.TEAL.ASSERT_FIELD_LENGTH, { exp: expected, got: val, line: line });
+    throw new TealError(ERRORS.TEAL.ASSERT_LENGTH, { exp: expected, got: val, line: line });
+  }
+}
+
+/**
+ * Description: assert index bound
+ * @param val Comparsion result
+ * @param expected expected result
+ * @param line Line number in TEAL file
+ */
+export function assertBound (len: number, idx: number): void {
+  if (idx >= len) {
+    throw new TealError(ERRORS.TEAL.INDEX_OUT_OF_BOUND);
+  }
```

Comment:
We have a function in `Op` ( i think `checkIndexBound`). Can you check if that can be used here?

Reply:
yes